src/calliope/schemas/config_schema.py

```diff
@@ -151,7 +164,7 @@ class SolveSpores(CalliopeBaseModel):
     """
 
     skip_baseline_run: bool = Field(default=False)
-    """If the model already contains `plan` mode results, use those as the initial base run results and start with SPORES iterations immediately."""
+    """If the model already contains `plan` results, use those as the initial base run results and start with SPORES iterations immediately."""
```

Comment:
```suggestion
    """If the model already contains `base` results, use those as the initial base run results and start with SPORES iterations immediately."""
```

Right? We've now removed all reference to `plan` in the docs at this point I think